### PR TITLE
Fix prebid timeout editable

### PR DESCRIPTION
**Pull request by AntoineJac**
public should be the field timeoutMillis so it could be changed if needed from the publisher code when integrating the SDK:
"PrebidMobile.timeoutMillis = 1000;"

I removed public for the var "TIMEOUT_MILLIS" which is final so public is useless.
Issue   #111

### CODE REVIEW

**Comment by AntoineJac** [comment]
@yoalex5 , for review. Thanks,

**Comment by AntoineJac** [comment]
@ppuviarasu , can you please review this PR? Thanks,

File: PrebidMobile/API1.0/src/main/java/org/prebid/mobile/PrebidMobile.java [hunk 1]
```diff
@@ -22,9 +22,9 @@
 
 public class PrebidMobile {
 
-    public static final int TIMEOUT_MILLIS = 2_000;
+    static final int TIMEOUT_MILLIS = 2_000;
```
Comment (yoalex5) [inline PrebidMobile/API1.0/src/main/java/org/prebid/mobile/PrebidMobile.java]: In this case we can use `private static final`

Comment (AntoineJac) [reply PrebidMobile/API1.0/src/main/java/org/prebid/mobile/PrebidMobile.java]: I agree, I have just made the change

File: PrebidMobile/API1.0/src/main/java/org/prebid/mobile/PrebidMobile.java [hunk 2]
```diff
@@ -22,9 +22,9 @@
 
 public class PrebidMobile {
 
-    public static final int TIMEOUT_MILLIS = 2_000;
+    private static final int TIMEOUT_MILLIS = 2_000;
 
-    static int timeoutMillis = TIMEOUT_MILLIS; // by default use 10000 milliseconds as timeout
+    public static int timeoutMillis = TIMEOUT_MILLIS; // by default use 10000 milliseconds as timeout
```
Comment (avohraa) [inline PrebidMobile/API1.0/src/main/java/org/prebid/mobile/PrebidMobile.java]: If the objective here is to providing publisher's code access to the `timeoutMillis`, then why not expose it using the public facing API. 
For reference:
```
    public static int getTimeoutMillis() {
        return timeoutMillis;
    }

    public static void setTimeoutMillis(int timeoutMillis) {
        PrebidMobile.timeoutMillis = timeoutMillis;
    }
```

Also, please correct the default value written in this comment - `// by default use 10000 milliseconds as timeout`

Comment (AntoineJac) [reply PrebidMobile/API1.0/src/main/java/org/prebid/mobile/PrebidMobile.java]: Hi @avohraa , I have made the change. Thanks,

**Pull request merged**